app/profile/stats/page.tsx

```diff
@@ -0,0 +1,144 @@
+"use client"
+
+import { useState, useEffect } from "react"
+import { Button } from "@/components/ui/button"
+import { useRouter } from "next/navigation"
+import Image from "next/image"
+import { Info } from "lucide-react"
+import { ProfileAPI } from "../api"
+import type { UserStats } from "../api/api-user-stats"
+
+export default function StatsPage() {
+  const router = useRouter()
+  const [userStats, setUserStats] = useState<UserStats>({
+    buyCompletion: { rate: "N/A", period: "(30d)" },
+    sellCompletion: { rate: "N/A", period: "(30d)" },
+    avgPayTime: { time: "N/A", period: "(30d)" },
+    avgReleaseTime: { time: "N/A", period: "(30d)" },
+    tradePartners: 0,
+    totalOrders30d: 0,
+    totalOrdersLifetime: 0,
+    tradeVolume30d: { amount: "0.00", currency: "USD", period: "(30d)" },
+    tradeVolumeLifetime: { amount: "0.00", currency: "USD" },
+  })
+  const [isLoading, setIsLoading] = useState(true)
+  const [error, setError] = useState<string | null>(null)
+
+  useEffect(() => {
+    const loadUserStats = async () => {
+      try {
+        setIsLoading(true)
+        setError(null)
+        const stats = await ProfileAPI.UserStats.fetchUserStats()
+        setUserStats(stats)
+      } catch (error) {
+        setError(error instanceof Error ? error.message : "Failed to load stats")
+      } finally {
+        setIsLoading(false)
+      }
+    }
+
+    loadUserStats()
+  }, [])
+
+  const handleBack = () => {
+    router.push("/profile")
+  }
+
+  const StatItem = ({
+    label,
+    value,
+    hasInfo = false,
+    showBorder = true,
+  }: {
+    label: string
+    value: string | number
+    hasInfo?: boolean
+    showBorder?: boolean
+  }) => (
+    <div className={`flex justify-between items-center w-full py-3 ${showBorder ? "border-b border-gray-200" : ""}`}>
+      <div className="flex items-center gap-1">
+        <span className="text-gray-600 text-sm">{label}</span>
+        {hasInfo && <Info className="h-4 w-4 text-gray-400" />}
```

Comment:
`nit`: same sizing for height and width can be written like this:

```suggestion
        {hasInfo && <Info className="size-4 text-gray-400" />}
```